ThreeBotPackages/tfgrid/workloads/models/tfgrid_workloads_reservation_actionable_1.toml

```diff
@@ -0,0 +1,3 @@
+
+@url = tfgrid.workloads.reservation_actionable.1
+reservation_id** = (I)
```

Comment:
Can this be 'workload_id` as explained above, this will make the logic much easier, and also generic because this action queue doesn't only have the 'to-delete' reservations but also, the ones that were waiting on the signing! This can also has the `node_id** = (S)` so you can find workloads by node directly. so you don't have to load the entire queue for each node poll request.